modules/components/src/utils/uri/__tests__/uri.test.js

```diff
@@ -1,6 +1,4 @@
-/* @flow */
-
-import { parseIntParam, parseSQONParam } from '..';
+import { parseIntParam, parseSQONParam } from '@/utils/uri';
```

Comment:
tests being executed from the monorepo's root are unable to handle the relative path correctly.